pkg/commands/run.go

```diff
@@ -113,6 +115,22 @@ func (r *RunCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.Bui
 	return nil
 }
 
+// addDefaultEnvs adds default values of variables (like $HOME) if they aren't already set
+func addDefaultEnvs(user string, envs []string) []string {
+	for _, env := range envs {
+		split := strings.SplitN(env, "=", 2)
+		if split[0] == constants.HOME {
+			return envs
```

Comment:
This seems like it can only set the default HOME, as designed. Maybe rename the function, or refactor it to support more default values in the future?